### PR TITLE
📝 add row-splitting to the tutorial

### PR DESCRIPTION
**Pull request by fiendish**

### CODE REVIEW

File: docs/source/tutorial/extract.rst [hunk 1]
```diff
@@ -345,6 +350,86 @@ The resulting intermediate output will look like:
...
+    )
+
+This says take "Use the ``specimens`` column as input, split any ``,`` or ``;``
+delimited values apart into their own entries, and then output the result to a
+standard concept column for biospecimen ID."
+
+.. note::
+
+    We use a special ``Split()`` object for lists of values that we want to
+    split apart. That lets us preserve the ability to return a list of values
```
Comment (znatty22) [inline docs/source/tutorial/extract.rst]:
```suggestion
    split apart into multiple rows. We could have split the string into a list and then used the existence of a list as the signal to split a row into multiple rows. However, we wanted to leave the option for a cell value to contain a list. This is why we use the Split object, which explicitly tells the extract stage to split a list of values into multiple rows.
```

File: docs/source/tutorial/extract.rst [hunk 2]
```diff
@@ -345,6 +350,86 @@ The resulting intermediate output will look like:
...
+    pertaining to a single entity without splitting them apart if we need to.
+
+.. note::
```
Comment (znatty22) [inline docs/source/tutorial/extract.rst]: Not sure if this note is necessary

File: docs/source/tutorial/extract.rst [hunk 3]
```diff
@@ -345,6 +350,86 @@ The resulting intermediate output will look like:
...
+
+.. code-block:: python
+
+    value_map(
+        in_col="specimens",
+        m=lambda x: Split(re.split('[,;]', x)),
+        out_col=CONCEPT.BIOSPECIMEN.ID
+    )
+
+This says take "Use the ``specimens`` column as input, split any ``,`` or ``;``
```
[the remark this answers is not in the document]
Comment (fiendish) [reply docs/source/tutorial/extract.rst]: 👌

File: docs/source/tutorial/extract.rst (hunk 1 above)
Comment (fiendish) [reply docs/source/tutorial/extract.rst]: 〰️

File: docs/source/tutorial/extract.rst (hunk 2 above)
Comment (fiendish) [reply docs/source/tutorial/extract.rst]: 👍

**Pull request merged**